Components/Snackbar/snackbar.html

```diff
@@ -117,6 +126,15 @@
                     </button>
                     <i style="color: white;" class="far fa-times-circle fa-lg"></i>
                 </div>
+                <button class="btn leadingdemo primary">Leading Demo</button>
+                <div class="snackbar success">
+                    <span class="snackbar_text-success">Item Uploaded Successfully
+                    </span>
+                    <button class="snackbar-btn">
+                       <b> UNDO </b>
+                    </button>
+                    <i style="color: white;" class="far fa-times-circle fa-lg"></i>
```

Comment:
For all the icon tags remove the inline style and create a utility class for that.

Reply:
Done. Thanks for pointing it out